src/Symfony/Component/Yaml/Yaml.php

```diff
@@ -29,6 +29,7 @@ class Yaml
     const DUMP_OBJECT_AS_MAP = 64;
     const DUMP_MULTI_LINE_LITERAL_BLOCK = 128;
     const PARSE_CONSTANT = 256;
+    const DUMP_EMPTY_ARRAY_AS_SEQUENCE = 512;
```

Comment:
I like the way it is now, the more recent addition are last with a higher number. I don't see why we should order them.